File: Code/Tier2/Common/src/main/java/javaModel/HttpResponse.java
package javaModel;

public class HttpResponse {

    private Boolean dbConfirmation;
    private String dbErrorString;

    public Boolean getDbResponse() {
        return dbConfirmation;
    }

    public void setDbResponse(Boolean dbResponse) {
        this.dbConfirmation = dbResponse;

    }

    public HttpResponse() {
        dbConfirmation=false;
        dbErrorString="";
    }

    public Boolean getDbConfirmation() {
        return dbConfirmation;
    }

    public void setDbConfirmation(Boolean dbConfirmation) {
        this.dbConfirmation = dbConfirmation;
    }
}

File: Code/Tier3-DatabaseLayer/RestServer/Common/HttpResponse.cs
using System;

namespace WebApplication.Common
{
    public class HttpResponse
    {
        public Boolean dbConfirmation { get; set; }
        public string dbErrorString { get; set; }

        public HttpResponse()
        {
            dbConfirmation = false;
            dbErrorString = "";
        }
        
        public HttpResponse(Boolean dbConfirmation)
        {
            this.dbConfirmation = dbConfirmation;
            dbErrorString = "";
        }
        public HttpResponse(Boolean dbConfirmation,string dbErrorString)
        {
            this.dbConfirmation = dbConfirmation;
            this.dbErrorString = dbErrorString;
        }
    }
}